### PR TITLE
tsc: add 20231109_TSC_MOM

### PR DESCRIPTION
**Pull request by ebail**

### CODE REVIEW

File: TSC_Minutes/20231109_TSC_MOM.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# SEAPATH -  TSC Meeting
+
+* Date: 12th October 2023
```
Comment (dupremathieu) [inline TSC_Minutes/20231109_TSC_MOM.md:3]: 9th November

File: TSC_Minutes/20231109_TSC_MOM.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        - worried with vendor-lockin (ex: Vmware)
+        - VPAC
+            - Interrested to implement virtualized protection
+    - Could be interresting to showcase SEAPATH next IEC weeks
+
+- Presentation of IEC62443 
+- Presentation available at:
+    - https://github.com/com-pas/contributing/blob/main/presentations/231109%20CoMPAS%20introduction%20for%20Seapath.pptx
+    - https://github.com/seapath/TSC/blob/master/TSC_Minutes/20231012_WelotecIEC62443.pdf
+
```
Comment (dupremathieu) [inline TSC_Minutes/20231109_TSC_MOM.md:40]: The presentation of the IEC 62443-4-2 is missing.

**Review by dupremathieu — CHANGES_REQUESTED** [review]

File: TSC_Minutes/20231109_TSC_MOM.md (hunk 1 above)
Comment (ebail) [reply TSC_Minutes/20231109_TSC_MOM.md:3]: good catch, thanks

File: TSC_Minutes/20231109_TSC_MOM.md (hunk 2 above)
Comment (ebail) [reply TSC_Minutes/20231109_TSC_MOM.md:40]: good catch

**Comment by ebail** [comment]
@dupremathieu

**Pull request merged**